sc2reader/data/ability_lookup.csv

```diff
@@ -1,18 +1,18 @@
 250mmStrikeCannons,250mmStrikeCannons,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,Cancel250mmStrikeCannons,
-ArchonWarp,ArchonWarpSelection,ArchonWarpTarget,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+ArchonWarp,SelectedUnits,ArchonWarpTarget,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 ArmSiloWithNuke,BuildNuke,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,CancelNuke,
-ArmoryResearch,,,UpgradeVehiclePlating1,UpgradeVehiclePlating2,UpgradeVehiclePlating3,UpgradeVehicleWeapons1,UpgradeVehicleWeapons2,UpgradeVehicleWeapons3,UpgradeShipPlating1,UpgradeShipPlating2,UpgradeShipPlating3,UpgradeShipWeapons1,UpgradeShipWeapons2,UpgradeShipWeapons3,,,,,,,,,,,,,,,,,CancelArmoryResearch,
+ArmoryResearch,,,UpgradeVehiclePlating1,UpgradeVehiclePlating2,UpgradeVehiclePlating3,ResearchTerranVehicleWeaponsLevel1,ResearchTerranVehicleWeaponsLevel2,ResearchTerranVehicleWeaponsLevel3,UpgradeShipPlating1,UpgradeShipPlating2,UpgradeShipPlating3,ResearchTerranShipWeaponsLevel1,ResearchTerranShipWeaponsLevel2,ResearchTerranShipWeaponsLevel3,ResearchTerranVehicleAndShipArmorsLevel1,ResearchTerranVehicleAndShipArmorsLevel2,ResearchTerranVehicleAndShipArmorsLevel3,,,,,,,,,,,,,,CancelArmoryResearch,
 AssaultMode,AssaultMode,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-AttackRedirect,Attack,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+AttackRedirect,AttackRedirect,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 AttackWarpPrism,AttackWarpPrism,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-BanelingNestResearch,EvolveCentrifugalHooks,EvolveTunnelingJaws,,,,,,,,,,,,,,,,,,,,,,,,,,,,,CancelBanelingNestResearch,
-BansheeCloak,CloakBanshee,DecloakBanshee,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+BanelingNestResearch,ResearchCentrificalHooks,EvolveTunnelingJaws,,,,,,,,,,,,,,,,,,,,,,,,,,,,,CancelBanelingNestResearch,
```

Comment:
Overwritten old command name with 'research' prefix rather than 'evolve' prefix. This reflects a change in the structure of the balance data export.